packages/sdk-govern/scripts/executeGovernorDomain.ts

```diff
@@ -0,0 +1,52 @@
+import * as config from '@nomad-xyz/configuration';
+import {NomadContext} from "@nomad-xyz/sdk";
+import {getCallBatch, getConfig} from './utils';
+import {CallBatch, CallBatchContents} from "../dist";
+import * as dotenv from 'dotenv';
+import {equalIds} from "@nomad-xyz/multi-provider/dist/utils";
+import * as ethers from "ethers";
+dotenv.config();
+
+run();
```

Comment:
if we're adding 3 extra CLIs it's time to just suck it up and write a [commander](https://www.npmjs.com/package/commander) cli

Reply:
https://github.com/nomad-xyz/monorepo/issues/243